src/components/tr/tr-styles.jsx

```diff
@@ -9,17 +9,17 @@ export default theme => {
       ...mixins.basicBoxSizing,
       ...styleUtils.flexRow(),
       ...styleUtils.sizes(),
-      alignItems: 'center',
       borderLeft: '1px solid transparent',
       borderRight: '1px solid transparent',
+      flexWrap: 'wrap',
 
       '&.primary': {
-        background: `${color.grayLight} !important`,
+        background: () => `${color.grayLight}`,
```

Comment:
Why add the fat arrow? Is it to get it more specific?

Ideally this should be `backgroundColor: color.grayLight`

Reply:
Fixed